src/cnlpt/cnlp_data.py

```diff
@@ -744,7 +739,7 @@ def _reconcile_labels_lists(self, processor):
                 new_labels = set(labels)
                 old_labels = set(self.tasks_to_labels[task])
                 if new_labels.isdisjoint(old_labels):
-                    raise Exception("The same task name has disjoint sets of labels in different dataset: %s vs. %s" % (str(str(old_labels), str(new_labels))))
+                    raise Exception("The same task name has disjoint sets of labels in different dataset: %s vs. %s" % (str(old_labels), str(new_labels)))
```

Comment:
@tmills can you confirm this is fixing something incorrect and not that i'm just misunderstanding your code here

Reply:
Lol when you really want to be sure it's a string